docs/guides/how_to_guides/configuring_datasources/how_to_configure_a_mssql_datasource.rst

```diff
@@ -118,7 +118,7 @@ Steps
         .. admonition:: Prerequisites: This how-to guide assumes you have already:
 
             - :ref:`Set up a working deployment of Great Expectations <tutorials__getting_started>`
-            - :ref:`Understand the basics of ExecutionEnvironments <execution_environments>`
+            - :ref:`Understand the basics of Datasources <datasources>`
             - Learned how to configure a :ref:`DataContext using test_yaml_config <how_configure_data_context_using_test_yaml_config>`
```

Comment:
similar issue with this link : 

`_how_to_guides_how_to_configure_datacontext_components_using_test_yaml_config` is the one you want

Reply:
as a separate note, I just found something I missed with this file: 

there is a duplicate `prerequisites` section, one in the tabs, and one above them. So sorry for missing this 😓

Reply:
Fixing

Reply:
one last tweak for the links to work  `_` character  needs to be removed from the beginning (so that it's `<reference_core_concepts_datasources>`

Reply:
with a similar change needing to be made for the remaining references